command/operator_diagnose.go

```diff
@@ -110,7 +120,7 @@ func (c *OperatorDiagnoseCommand) RunWithParsedFlags() int {
 	}
 
 	c.UI.Output(version.GetVersion().FullVersionNumber(true))
-
+	rloadFuncs := make(map[string][]reloadutil.ReloadFunc, 5)
```

Comment:
does the `5` in your `make()` do anything here? my understanding is you don't include a length for a map. i was wondering if maybe there was some special case where that applies to the length of the slice value, but i tried it out in go playground and that doesn't seem to be the case.

Reply:
Good catch! I used that for testing to explicitly initialize to 5, and forgot to remove it.